Hosted billing documentation (#2403)
astronomer/docs: astro/manage-billing.md

---
sidebar_label: 'Manage billing'
title: 'Manage Astro billing'
id: manage-billing
description: "Change your billing details and view your current spend from the Cloud UI."
---

> On Astro Hosted, you're billed based on how many hours you run your Deployment, as well as how many resources your Deployment components use. See [Pricing](https://www.astronomer.io/pricing/) for more information.

Author: ```suggestion
On Astro Hosted, you're billed based on how many hours you run your Deployment, as well as how many resources your Deployment components use. See [Pricing](https://www.astronomer.io/pricing/) for more a complete breakdown of possible costs.
```

> You can configure payment information and check your total Astro spend from the Cloud UI.

Author: ```suggestion
You can configure payment information and check your total Astro spend from the Cloud UI so that you don't go over your budget for running Airflow.
```

> ## Update billing details

Author: ```suggestion
## View billing details
```

- Each **Charge** is an Astro component that you used at some point in the billing cycle. Note that your worker resources are charged separately from the Deployments they run on and each other. For example, a Deployment with two worker queues will appear as three separate charges in your invoice. 
- **Quantity** represents the number of hours you ran a given Astro component.

:::info

The **Billing** page updates every hour with your total spend in the previous complete hour. For example, at 10:00 AM, the page updates with your spend from 8:00 AM - 9:00 AM on that same day. If your spending metrics don't look accurate based on recent usage, wait until the next complete hour for the page to update.

:::

## View component usage by date

1. In the Cloud UI, click Astronomer logo in the upper left corner to open your Organization page. Then, click **Settings** > **Billing**. 
2. Click **Usage**. 

Each chart on this page shows your usage over the last 30, 60, or 90 days for each billable component type. Hover over nodes on the chart to see your total hour spend for a component on a given day. Use the dropdown menu in the upper right corner to change the timeframe for each chart.

> The charts are organized by component type. For a given component type, the chart shows usage for all instances of a given component across your Organization. Therefore, it's possible to have more than 24 hours of usage for a given day. For example, if you have two **Deployment with Medium Schedulers** running for 24 hours in a day, the usage chart will show that your total usage for the day was 48 hours.

Author: ```suggestion
The charts are organized by component type. For a given component type, the chart shows usage for all instances of a given component type across your Organization. Therefore, it's possible to have more than 24 hours of usage for a given day. For example, if you have two **Deployment with Medium Schedulers** running for 24 hours in a day, the usage chart will show that your total usage for the day was 48 hours.
```
